tests/test_server/fixtures/factories/operation.py

```diff
@@ -71,9 +72,15 @@ async def operation(
     params = request.param
 
     async with async_session_maker() as async_session:
+        sql_query = await create_sql_query(async_session=async_session)
```

Comment:
IMHO sql_query_id should be passed explicitly via factory params, and not created for every operation

Reply:
Do you mean create separate tests only to check response with sql_query?